cfn-resources/stream-instance/template.yml

```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: AWS SAM template for the MongoDB::Atlas::StreamInstance resource type
+
+Globals:
+  Function:
+    Timeout: 180 # docker start-up times can be long for SAM CLI
+    MemorySize: 256
+
+Resources:
+  TypeFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: bootstrap
+      Runtime: provided.al2
+      CodeUri: bin/
+
+  TestEntrypoint:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: bootstrap
+      Runtime: provided.al2
+      CodeUri: bin/
+      Environment:
+        Variables:
+          MODE: Test
+          LOG_LEVEL: debug
+          MONGODB_ATLAS_BASE_URL: https://cloud-dev.mongodb.com/ 
```

Comment:
nit: I believe this env variable is not needed because the base url is being set in the AWS secret coming from the profile.

Reply:
I think having the prop here is a good addition as it prevents running the test against prod if the profile does not have the base url defined

Reply:
should it be cloud-dev though?

Reply:
Different to TF, for CFN we do not make use of env variables to define the base url, we rely exclusively on the value defined in the profile.